spec/integration/submit_token_url_spec.rb

```diff
@@ -37,6 +37,13 @@
     expect(page).to have_content 'Email is invalid'
   end
 
+  scenario 'will contain all the form fields', js: true do
+    visit request_access_notice_path(notice)
+
+    expect(page).to have_content 'Email address'
+    expect(page).to have_content 'Select to get a notification when new notice documents are added (or when exisiting notice documents are updated)'
```

Comment:
Typo fix, please: `Select to get a notification when new notice documents are added (or when existing notice documents are updated).`

Reply:
Done